packages/react/src/components/StructuredList/StructuredList.js

```diff
@@ -127,32 +126,38 @@ StructuredListBody.defaultProps = {
   onKeyDown: () => {},
 };
 
+function useFocusWithin() {
```

Comment:
If we want to make this into its own hook we should totally implement this / move over to `internal`. Otherwise if we just want it in this component for now we could totally just remove it 👍